standards/RDA/ruleset.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<ruleset name="RDA">
+	<description>Reader's Digest Association WordPress Coding Standards</description>
+
+	<rule ref="WordPress-Core">
+		<!-- This rule is foolish and breaks many clean code uses -->
+		<exclude name="PEAR.Functions.FunctionCallSignature.Indent" />
+		<exclude name="WordPress.PHP.YodaConditions" />
+		<exclude name="WordPress.WP.GlobalVariablesOverride" />
+		<!-- This rule is foolish and breaks many legitimate uses -->
+		<exclude name="WordPress.Security.EscapeOutput.OutputNotEscaped" />
```

Comment:
While this is annoying, it's also a security concern. If we exclude the sniff, we should replace it with some other way to enforce security when printing potentially unsafe code.